bin/docs.test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ev
+
+# Make sure we use the correct repo root dir
+DIR=`dirname $0`
+REPO_ROOT=$DIR/..
+pushd $REPO_ROOT >/dev/null
+rm -rf sandbox/loopback.io/
+# Shadow clone is faster
+git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
+./node_modules/lerna/bin/lerna.js bootstrap --scope loopback.io-workflow-scripts
+pushd $REPO_ROOT/sandbox/loopback.io/ >/dev/null
+bundle install
+npm run build
```

Comment:
Should we record the exit code of the command above and exit the script with it? Otherwise, can travis still detect the failure?

Reply:
See my comment above with regards to the `set -ev`. If my understanding about the flag is incorrect please let me know. I did make commits to this PR to verify the following and they all worked as expected:

- `loopback.io` stage exits early with success if no changes to `@loopback/docs`
- `loopback.io` stage builds successfully with changes to `@loopback/docs`
- `loopback.io` stage fails if a change to `@loopback/docs` will break Jekyll